Libplanet.Explorer/Queries/BlockQuery.cs

```diff
@@ -43,13 +43,30 @@ public BlockQuery()
             Field<BlockType<T>>(
                 "block",
                 arguments: new QueryArguments(
-                    new QueryArgument<IdGraphType> { Name = "hash" }
+                    new QueryArgument<IdGraphType> { Name = "hash" },
+                    new QueryArgument<IdGraphType> { Name = "index" }
                 ),
                 resolve: context =>
                 {
-                    HashDigest<SHA256> hash = HashDigest<SHA256>.FromString(
-                        context.GetArgument<string>("hash"));
-                    return Query<T>.GetBlock(hash);
+                    string? hash = context.GetArgument<string?>("hash", null);
```

Comment:
[Nullable reference types] is the feature in C# 8.0. Actually `string` type can be null without nullable.

```suggestion
                    string hash = context.GetArgument<string>("hash");
```

```
// Libplanet.Explorer/Queries/BlockQuery.cs#L51

Feature 'nullable reference types' is not available in C# 7.2. Please use language version 8.0 or greater.
```

[Nullable reference types]: https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references